packages/client/base/src/types/hosted/v3/CrossmintHostedCheckoutV3Props.ts

```diff
@@ -36,8 +36,19 @@ export interface CrossmintHostedCheckoutV3Appearance {
     variables?: CrossmintHostedCheckoutV3AppearanceVariables;
     overlay?: CrossmintHostedCheckoutV3OverlayOptions;
     display?: "popup" | "new-tab" | "same-tab";
+    rules?: HostedCheckoutV3AppearanceRules;
 }
 
+// TODO: Should we just allow all the other embedded rules?
```

Comment:
I wonder if we should just reuse the existing interface from embedded. Any reason not to do it?